packages/viewer/src/scenes/scene.ts

```diff
@@ -62,6 +64,53 @@ export class SceneItemOperationsBuilder
     );
   }
 
+  public transform(
+    matrix: vertexvis.protobuf.core.IMatrix4x4f | number[]
```

Comment:
what are the cases for allowing either option for specifying the matrix? does something provide the transform in that way where we want to just pass through without modifying?

Reply:
I think we should keep the API client more closely in line with what's expected in the protos to minimize additional logic, and move this conversion step a level higher into the viewer operations.

Reply:
Yeah, I suppose we don't necessarily have to support the `number[]` option here, my thinking was to have a slightly easier option for specifying this transform when using the SDK over CDN. Somewhat similar to the change we made to `materialOverride` to support a hex string (though not quite as useful here)